src/database/config.js

```diff
@@ -0,0 +1,21 @@
+import pg from "pg";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const isProduction = process.env.NODE_ENV === "production";
+
+/**
+ * Use this pattern in naming your local connection strings
+ *  postgres://{database_username}:{database_password}@{host}:{port}/{database_name}
+ */
+
+const connectionString = isProduction
+  ? process.env.DATABASE_URL
+  : "postgres://postgres:juninho1@127.0.0.1:5432/weTravel";
```

Comment:
Pass the alternative DB url in to the environment variable. You are exposing db secret this way

Reply:
yes, if you want to do it this way, you'd have to pass the second db url as an environment variable. however, there's another way to do it, at every instance you may not really need more than one db URL. So we can just use one environment variable DATABASE_URL.
on your local machine you'll set this up to point to your local database, on travis it'll point to travis postgres db and on heroku it'll point to heroku postgres db.
what do you think?

Reply:
I think @victor-nach's suggestion is great. However, the last time I tried that I couldn't get it to work properly. It is possible the travis doesn't get dropped after the test fails or passes, which leads to errors. @victor-nach is this the case, if  not share some insight into setting up and using travis db

Reply:
@debelistic the env PR have not been merged. So I had to pass the local db url so that it can be tested when you are testing the connection